codecov_cli/services/commit/__init__.py

```diff
@@ -44,18 +44,22 @@ def create_commit_logic(
 def send_commit_data(
     commit_sha, parent_sha, pr, branch, slug, token, service, enterprise_url
 ):
+    decoded_slug = decode_slug(slug)
+    pull_dict = get_pull(service, decoded_slug, pr) if not token else None
+    if is_fork_pr(pull_dict):
+        headers = {}
+        branch = pull_dict["head"]["slug"] + ":" + branch
```

Comment:
i think this is good here, just a note for later:

since `:` is a legal character in real branch names, i think in the API when we double-check that this PR is from a fork we should set a different field entirely in the database that the UI can check instead of trying to parse it out of the branch name (see ugly screenshot mockup)

![Screenshot 2023-11-13 at 10 40 40 AM](https://github.com/codecov/codecov-cli/assets/137832199/530e238a-c6be-4b35-949e-71817bf282db)

Reply:
are you sure it's a valid character? I tried to create a branch with a ":" but says it's invalid, and the documentations say that too. see [here](https://git-scm.com/docs/git-check-ref-format) and [here](https://docs.github.com/en/get-started/using-git/dealing-with-special-characters-in-branch-and-tag-names#about-branch-and-tag-names)

Reply:
but would be a nice UI addition for PRs coming from a fork

Reply:
oh i stand corrected, you're right! i should have checked official docs, i just read some page that said git branch names were "very permissive"